docs/swagger-v2.yaml

```diff
@@ -592,7 +648,7 @@ components:
           description: 'The Id of the session. Example: "123123".'
         merchantSerialNumber:
           type: string
-          description: 'The merchant serial number for the sales unit. Example: "59474"'
+          description: 'The merchant serial number for the sale unit. Example: "59474"'
```

Comment:
not sure this is wrong 🤔

Reply:
I'll check!

Reply:
Seems like 'sale unit' won the argument here: https://vippsas.slack.com/archives/C8HP1AJ79/p1677672538118429

Reply:
Sales unit is used in the portal which is where the consumer will encounter the term. So for that reason I would say sales unit is better.
(This may be one of the most frequently debated topics in Vipps, IMO any variant works)